### PR TITLE
Feature/replace jquery tabs

### PR DESCRIPTION
**Pull request by p0psicles**
Goals for this PR:
* Remove all occurrences of the jquery-ui tabs, with a custom code.

Additionally i did some CSS fixes. But i'll get these out of here, and put them in a separate PR.

### CODE REVIEW

File: gui/slick/css/style.css [hunk 1]
```diff
@@ -180,7 +188,7 @@ inc_top.mako
 
 #content {
     width: 95%;
-    min-width: 875px;
+    /*min-width: 875px;*/
```
Comment (p0psicles) [inline gui/slick/css/style.css]: This does fix most of the non-responsiveness.

File: gui/slick/views/layouts/main.mako [hunk 2]
```diff
@@ -48,14 +48,18 @@
     </head>
     <body ${('data-controller="' + controller + '" data-action="' + action + '" api-key="' + sickbeard.API_KEY +'"  api-root="api/v2/"', '')[title == 'Login']}>
         <%include file="/partials/header.mako"/>
-        % if submenu:
-        <%include file="/partials/submenu.mako"/>
-        % endif
-        <%include file="/partials/alerts.mako"/>
+        
         <div id="contentWrapper">
+        <div class="row">
```
Comment (p0psicles) [inline gui/slick/views/layouts/main.mako]: Was playing with aligning the submenu and the main content. As the main content always has a left and right margin. While I think the submenu should also have this.

File: gui/slick/views/partials/submenu.mako [hunk 3]
```diff
@@ -1,26 +1,27 @@
...
-                  % endfor
-              % else:
-                  <a href="${menuItem['path']}" class="btn${('', ' confirm ' + menuItem.get('class', ''))['confirm' in menuItem]}">${('', '<span class="pull-left ' + icon_class + '"></span> ')[bool(icon_class)]}${menuItem['title']}</a>
-                  <% first = False %>
-              % endif
-        % endif
-    % endfor
-    </span>
-</div>
+
```
Comment (p0psicles) [inline gui/slick/views/partials/submenu.mako]: Better allignment of the submenu. Ill remove this, and put it in a separate PR.

File: gui/slick/js/ui-tabs.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
+$.fn.uiTabs = function(defaultTab) {
+    // http://stackoverflow.com/a/9282379/2311366
+    (function(namespace) {
```
Comment (fernandog) [inline gui/slick/js/ui-tabs.js]: @OmgImAlexis what about this one ?

**Comment by p0psicles** [comment]
Are we still interested in replacing the UI tabs? Else I'm going to copy the other UI fixes from this one,  copy it to new pr,  and close this Pr. @omg.

**Comment by OmgImAlexis** [comment]
Yeah, if we can. This would be easier to integrate into vue than jQuery's one. If there's no issues can the other fixes be added as a PR and then the tabs themselves be added?

I'll open a PR to add all the new APIv2 stuff which means we can start working on vue.

File: gui/slick/css/style.css (hunk 1 above)
Comment (p0psicles) [inline gui/slick/css/style.css]: Will go in new pr if still needed.
